postgresql/endpoint_methods.go

```diff
@@ -0,0 +1,56 @@
+package postgresql
+
+import (
+	"github.com/lib/pq"
+
+	"github.com/benchttp/server/benchttp"
+)
+
+func (c *computedStatsService) ListMetadataByUserID(userID int) ([]*benchttp.Metadata, error) {
+	metadataList := make([]*benchttp.Metadata, 0)
+
+	stmt, err := c.db.Prepare("SELECT tag, finished_at FROM metadata WHERE user_id = $1 ORDER BY finished_at DESC")
+	if err != nil {
+		return []*benchttp.Metadata{}, ErrPreparingStmt
+	}
+	defer stmt.Close()
+
+	rows, err := stmt.Query(userID)
+	if err != nil {
+		return []*benchttp.Metadata{}, ErrExecutingPreparedStmt
+	}
+	defer rows.Close()
+
+	for rows.Next() {
+		metadata := new(benchttp.Metadata)
+		err = rows.Scan(
+			&metadata.Tag,
+			&metadata.FinishedAt,
+		)
+		if err != nil {
+			return nil, err
+		}
+		metadataList = append(metadataList, metadata)
+	}
+
+	return metadataList, nil
+}
+
+func (c *computedStatsService) FindComputedStatsByID(metadataID int) (*benchttp.ComputedStats, error) {
+	computedStats := &benchttp.ComputedStats{}
+
+	stmt := "SELECT m.tag, m.finished_at, c.code_1xx, c.code_2xx, c.code_3xx, c.code_4xx, c.code_5xx, t.min, t.max, t.mean, t.median, t.variance, t.deciles " +
+		"FROM public.metadata AS m " +
+		"INNER JOIN public.codestats AS c ON c.metadata_id = m.id " +
+		"INNER JOIN public.timestats AS t ON t.metadata_id = m.id " +
+		"WHERE m.id = $1 " +
+		"ORDER BY m.finished_at DESC"
```

Comment:
A bit more friendly:
```suggestion
	stmt := `
SELECT
	m.tag,
	m.finished_at,
	c.code_1xx,
	c.code_2xx,
	c.code_3xx,
	c.code_4xx,
	c.code_5xx,
	t.min,
	t.max,
	t.mean,
	t.median,
	t.variance,
	t.deciles,
FROM public.metadata AS m
INNER JOIN public.codestats AS c ON c.metadata_id = m.id
INNER JOIN public.timestats AS t ON t.metadata_id = m.id
WHERE m.id = $1
ORDER BY m.finished_at DESC`[1:]
```
Or alternatively a SQL view 🙃 

Note: I also added missing `c.code_5xx`